Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp

```diff
@@ -190,7 +190,8 @@ GPUProcessProxy::GPUProcessProxy()
     }
 
     if (!containerTemporaryDirectory.isEmpty()) {
-        if (auto handle = SandboxExtension::createHandleWithoutResolvingPath(containerTemporaryDirectory, SandboxExtension::Type::ReadWrite))
+        auto tempDirectory = resolveAndCreateReadWriteDirectoryForSandboxExtension(makeString(containerTemporaryDirectory, "/com.apple.WebKit.GPU"_s));
```

Comment:
Is it really OK to hard-code the `/` here? Shouldn't we use the file system API to append the directory?

Reply:
Good point, will fix!

Thanks for reviewing!